templates/account/login.html

```diff
@@ -1,35 +1,35 @@
 {% extends "web/base.html" %}
-{% load static %}
+{% load static i18n %}
 
 {% block body %}
 <div id="fullpage-form-wrapper">
     <div id="fullpage-form">
-        <h1 class="title is-size-4">Sign in</h1>
+        <h1 class="title is-size-4">{% trans "Sign in" %}</h1>
         <form method="post">
             {% csrf_token %}
             {{ form.non_field_errors }}
             <div class="field">
-              <label class="label">Email</label>
+              <label class="label">{% trans "Email" %}</label>
               <div class="control">
                 {{ form.login }}
               </div>
               {{ form.login.errors }}
             </div>
             <div class="field">
-              <label class="label">Password</label>
+              <label class="label">{% trans "Password" %}</label>
               <div class="control">
                 {{ form.password }}
               </div>
               {{ form.password.errors }}
             </div>
-            <input class="button is-primary is-fullwidth" type="submit" value="Sign in">
+            <input class="button is-primary is-fullwidth" type="submit" value="{% trans "Sign in" %}">
```

Comment:
nit: I like using single quotes for such places so its less confusing to watch and also avoids this being highlighted as an error in the code.
`"{% trans 'Sign in' %}"`